### ISSUE TITLE
SDK 33 - Lottie warning

### ISSUE THREAD

**Issue by Drusy** [expo/expo#4455]
### Environment

>   Expo CLI 2.19.1 environment info:
>     System:
>       OS: macOS 10.14.5
>       Shell: 5.0.7 - /usr/local/bin/bash
>     Binaries:
>       Node: 11.15.0 - ~/.nvm/versions/node/v11.15.0/bin/node
>       Yarn: 1.16.0 - /usr/local/bin/yarn
>       npm: 6.7.0 - ~/.nvm/versions/node/v11.15.0/bin/npm
>       Watchman: 4.9.0 - /usr/local/bin/watchman
>     IDEs:
>       Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
>     npmPackages:
>       @bugsnag/expo: ^6.3.0 => 6.3.0 
>       expo: ^33.0.0 => 33.0.3 
>       react: 16.8.3 => 16.8.3 
>       react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8 
>       react-navigation: ^3.0.9 => 3.9.2 
>     npmGlobalPackages:
>       expo-cli: 2.19.1

Target: iOS Simulator via `yarn start`

### Steps to Reproduce

Moving from 

```
import {DangerZone} from 'expo';
let {Lottie} = DangerZone;
```

to

```
import Lottie from 'lottie-react-native';
```

### Expected Behavior

No warning

### Actual Behavior

Shows the following warning:

> ReactNative.NativeModules.LottieAnimationView.getConstants did not have a corresponding prop defined
> in the mock provided to SafeModule.
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:67263:28 in warn
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:100590:23 in warn
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:108187:23 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:108185:34 in SafeModuleCreate
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:251990:65 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:251872:31 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:251794:62 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:250063:56 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:179958:55 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:154445:57 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:1385:48 in <unknown>
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:276:14 in loadModuleImplementation
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:152:47 in guardedLoadModule
> * http://10.1.0.190:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:287997:4 in global code
> 

### Reproducible Demo

Basic expo project, importing Lottie: https://github.com/Drusy/expo-sdk33-lottie-bugreport-warning.

`yarn install`
`expo start`

**Comment by ken0x0a** [comment]
I got same warning

**Comment by changLiuUNSW** [comment]
+1

**Comment by ismaelbarry** [comment]
I'm getting the same issue as well. Was wondering if you were able to find a solution.

ReactNative.NativeModules.LottieAnimationView.getConstants did not have a corresponding prop defined
in the mock provided to SafeModule.
- node_modules/expo/build/environment/muteWarnings.fx.js:17:23 in warn
- node_modules/react-native-safe-module/lib/NativeSafeModule.js:116:13 in <unknown>
- node_modules/react-native-safe-module/lib/NativeSafeModule.js:114:28 in SafeModuleCreate
- node_modules/lottie-react-native/lib/LottieView.js:1:2657 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
- node_modules/lottie-react-native/lib/index.js:1:22 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
* components/animations/SuccessAnimation.js:6:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
* screens/Authentication/LoginScreen.js:13:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
* navigation/MainTabNavigator.js:23:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
* navigation/AppNavigator.js:4:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
* App.js:8:0 in <unknown>

**Comment by mattiassamskar** [comment]
+1

**Comment by Alvaro97gm** [comment]
Same issue

**Comment by rjanas** [comment]
Occures on iOS only.

**Comment by rakeshpetit** [comment]
We need to change Lottie to LottieView to use the below imports.
import LottieView from 'lottie-react-native'

**Comment by davidbiller** [comment]
> We need to change Lottie to LottieView and use the below imports.
> import LottieView from 'lottie-react-native'

Didnt work for me

**Comment by tonnenpinguin** [comment]
> We need to change Lottie to LottieView and use the below imports.
> import LottieView from 'lottie-react-native'

Given that this just uses the default export anyway the naming you use for the import shouldn't matter afaik.

**Comment by thuann-vn** [comment]
Any solution for this? I'm facing it too :(

**Comment by eblancperso** [comment]
Until the problem is fixed I suppose you can just suppress the warning with:

import { YellowBox } from "react-native";

YellowBox.ignoreWarnings(["ReactNative.NativeModules.LottieAnimationView.getConstants"]);

**Comment by dereknelson** [comment]
added test case in their logger in [this pr](https://github.com/expo/expo/pull/4638)

**Comment by allhaze** [comment]
Same issue :(

**Comment by Yasernova** [comment]
same here

**Comment by KangYoosam** [comment]
same problem in my app

**Comment by stavros-zavrakas** [comment]
The solution that @dereknelson recommends works for me. His solution seems that already got merged so I suspect it will be solved in the next expo published version?

**Comment by santomegonzalo** [comment]
I had the same problem but I just updated expo to the latest version `33.0.7` and now works perfectly.

**Comment by dayaki** [comment]
@santomegonzalo Please how did you upgrade to 33.0.7, simply by changing the sdkVersion in app.json?

**Comment by santomegonzalo** [comment]
Yes @dayaki ... I was on version `33.0.0` already... so I just increased the version to `33.0.7` and then `yarn install`

**Comment by dayaki** [comment]
@santomegonzalo adding `33.0.7` to app.json ONLY didn't work. Is there any thing else you did?

**Comment by santomegonzalo** [comment]
sorry, not to app.json ... just to package.json

**Comment by dayaki** [comment]
@santomegonzalo thanks, it worked.

**Comment by jonstuebe** [comment]
getting an error saying that the expo version doesn't exist

**Comment by caio-ribeiro-pereira** [comment]
I'm facing the same issue on expo `34.0.2` with lottie-react-native `2.6.1`

**Comment by YoshiYo** [comment]
Same here on expo `34.0.3` and lottie-react-native `2.6.1`

**Comment by changLiuUNSW** [comment]
I can confirm the issue come back again after update to expo 34 @dereknelson

**Comment by brandtnewww** [comment]
I also experienced defect animatons & performance issues.

**Comment by superandrew** [comment]
Same here on expo 34.0.3 and lottie-react-native 2.6.1

**Comment by raheelshah31** [comment]
+1

**Comment by Galzk** [comment]
Until we get to the next stable, or Lottie fixes the issue, you can mute the warning like this

```
import { YellowBox } from "react-native";
// Remove this and YellowBox after this issue is resolved https://github.com/expo/expo/issues/4455
YellowBox.ignoreWarnings(["UIManager['getConstants']"]);

```

**Comment by mfy5013** [comment]
Issue happened as soon as i upgraded to sdk 34

**Comment by jdnichollsc** [comment]
Same here using SDK 34 and `lottie-react-native` 2.6.1

**Comment by ddedic** [comment]
+1

**Comment by EdmundMai** [comment]
+1

**Comment by pROFESOR11** [comment]
Same here, happened right after sdk 34 upgrade

**Comment by murat0** [comment]
Same here; `"expo": "^34.0.1",` & `"lottie-react-native": "~2.6.1",`

**Comment by jdnichollsc** [comment]
`FATAL EXCEPTION` from **Android** => https://github.com/expo/expo/issues/5559

**Comment by miversen** [comment]
Same here, happened after sdk 34 upgrade

**Comment by dereknelson** [comment]
everyone coming here to leave the comment "same here" - I understand it may be out of the scope of your ability for some of you to fix it and open a PR.

however. 

not only did I open a [PR](https://github.com/expo/expo/pull/4638) to solve this originally (which took me a total of 2 minutes, tops), I commented a solution for you to **implement yourself until the change lands** in expo's SDK. It would take you maybe 60 seconds more to implement that **very simple change** as it would to comment "+1". 

tldr: stop complaining and do something about it!

**Comment by mfy5013** [comment]
Muting a warning is not solving what's causing the warning. The reason I commented same here is to let people who may be interested in this know what caused this issue for me and maybe let them know that this many people are following this issue.  Its not complaining.

**Comment by viko42** [comment]
upgrade to `"lottie-react-native": "~3.2.1",`

**Comment by jdnichollsc** [comment]
@viko42 what Expo SDK are you using? there's an error and we can't update lottie => https://github.com/expo/expo/issues/5559

**Comment by viko42** [comment]
@jdnichollsc I got the same error than u, my bad!

**Comment by cruzach** [comment]
Hi everyone! @dereknelson took care of muting the warning (thank you 🙏), and we plan to upgrade `lottie-react-native` once we upgrade over RN 0.60 ([here's the issue for that](https://github.com/expo/expo/issues/5559)) so I'm going to close this issue
